Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.yml

```diff
@@ -0,0 +1,128 @@
+category: Database
+commonfields:
+  id: Generic SQL
+  version: -1
+configuration:
+- defaultvalue: MySQL
+  display: SQL DB
+  hidden: false
+  name: dialect
+  options:
```

Comment:
don't we want an `other` option, and then the user will configure the port manually?

Reply:
the user can put any port he want's but there are only 4 options for dialect (dbs)

Reply:
Since the param is a `single-select` let's leave it as it is.